app/src/main/res/layout/tracks_item.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+
+        <import type="android.view.View" />
+
+        <variable
+            name="viewHolder"
+            type="org.fossasia.openevent.app.core.track.list.viewholder.TracksViewHolder" />
```

Comment:
This won't be needed when you will have static method. And instead of adding those methods in ViewHolder, add these methods in Presenter